sgx/untrusted/include/ocall/ecdh_ocall.h

```diff
@@ -51,7 +49,7 @@ extern "C"
  *
  * @return 0 on success, 1 on failure
  */
-  int setup_socket_ocall(const char *server_host, int server_host_len,
+  int setup_socket_ocall(const char *server_host,
```

Comment:
This change to the setup_socket_ocall() API breaks the demo code, which uses it. You have to also change the specification of this OCALL in the .edl file. The problem there, is that we specify the size of the 'server_host' string as 'count = server_host_len' and the SDK uses this for memory allocation. Since it is 'const char *', I tried not specifying the size, but that didn't work. I also believe that you cannot use something like strlen() in the .edl. What is the error that passing the length parameter causes?